api/src/test/java/org/openmrs/api/db/hibernate/HibernateDiagnosisDAOTest.java

```diff
@@ -92,8 +92,13 @@ public void shouldGetDiagnosisById() {
 	}
 
 	@Test
-	public void shouldGetActiveDiagnoses() {
-		assertEquals(2, diagnosisDAO.getActiveDiagnoses(new Patient(2)).size()); 
+	public void shouldGetActiveDiagnosesWithFromDate() {
+		assertEquals(1, diagnosisDAO.getActiveDiagnoses(new Patient(2), new Date(0)).size()); 
+	}
+
+	@Test
+	public void shouldGetActiveDiagnosesWithoutFromDate() {
```

Comment:
@dkayiwa  Here is the tests that gets diagnosis when not supplied with a start date